.github/workflows/auto-assign-issue.yml

```diff
@@ -0,0 +1,17 @@
+name: Assign Issue to Author
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  auto-assign:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: 'Auto-assign issue creator'
+        uses: pozil/auto-assign-issue@v2
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          assignees: ${{ github.event.issue.user.login }} # Auto-assign the issue creator
```

Comment:
Hello @JCarlosR,

I regularly write a lot of issues, but I don't necessarily assign them to myself. I believe that in the context of this project, it's not essential to always assign issues. This makes it easier for other contributors to take them on.

Reply:
Sure, let me revert this part then